TestLibrary.test_initialize_windows fails on Arch Linux ARM 32-bit

```
_____________________ TestLibrary.test_initialize_windows ______________________
self = <tests.unit.test_library.TestLibrary testMethod=test_initialize_windows>
mock_ctypes = <MagicMock name='ctypes' id='1914886720'>
mock_find_library = <MagicMock name='find_library' id='1914886904'>
mock_open = <MagicMock name='open' id='1914887088'>

    @mock.patch('sys.platform', new='windows')
    @mock.patch('pylink.library.open')
    @mock.patch('os.remove', new=mock.Mock())
    @mock.patch('tempfile.NamedTemporaryFile', new=mock.Mock())
    @mock.patch('ctypes.util.find_library')
    @mock.patch('pylink.library.ctypes')
    def test_initialize_windows(self, mock_ctypes, mock_find_library, mock_open):
        """Tests creating a library on a Windows machine.

        Args:
          self (TestLibrary): the ``TestLibrary`` instance
          mock_ctypes (Mock): a mocked version of the ctypes library
          mock_find_library (Mock): mock for mocking the
            ``ctypes.util.find_library()`` call
          mock_open (Mock): mock for mocking the call to ``open()``

        Returns:
          ``None``
        """
        mock_windll = mock.Mock()
        mock_windll.__getitem__ = mock.Mock()

        mock_cdll = mock.Mock()
        mock_cdll.__getitem__ = mock.Mock()

        mock_ctypes.windll = mock_windll
        mock_ctypes.cdll = mock_cdll
        mock_find_library.return_value = self.lib_path

        lib = library.Library()
        lib.unload = mock.Mock()

>       mock_find_library.assert_called_once_with(library.Library.WINDOWS_64_JLINK_SDK_NAME)

tests/unit/test_library.py:254:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python3.14/unittest/mock.py:997: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <MagicMock name='find_library' id='1914886904'>, args = ('JLink_x64',)
kwargs = {}, expected = call('JLink_x64'), actual = call('JLinkARM')
_error_message = <function NonCallableMock.assert_called_with.<locals>._error_message at 0x722241e0>
cause = None

    def assert_called_with(self, /, *args, **kwargs):
        """assert that the last call was made with the specified arguments.

        Raises an AssertionError if the args and keyword args passed in are
        different to the last call to the mock."""
        if self.call_args is None:
            expected = self._format_mock_call_signature(args, kwargs)
            actual = 'not called.'
            error_message = ('expected call not found.\nExpected: %s\n  Actual:
%s'
                    % (expected, actual))
            raise AssertionError(error_message)

        def _error_message():
            msg = self._format_mock_failure_message(args, kwargs)
            return msg
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        actual = self._call_matcher(self.call_args)
        if actual != expected:
            cause = expected if isinstance(expected, Exception) else None
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: find_library('JLink_x64')
E             Actual: find_library('JLinkARM')

/usr/lib/python3.14/unittest/mock.py:985: AssertionError
=========================== short test summary info ============================
FAILED tests/unit/test_library.py::TestLibrary::test_initialize_windows - AssertionError: expected call not found.
Expected: find_library('JLink_x64')
  Actual: find_library('JLinkARM')
```

--- a/tests/unit/test_library.py
+++ b/tests/unit/test_library.py
@@ -216,6 +216,7 @@ class TestLibrary(unittest.TestCase):
         mock_load_library.assert_called_once()
 
     @mock.patch('sys.platform', new='windows')
+    @mock.patch('sys.maxsize', new=(2**63-1))
     @mock.patch('pylink.library.open')
     @mock.patch('os.remove', new=mock.Mock())
     @mock.patch('tempfile.NamedTemporaryFile', new=mock.Mock())

